server/py/services/api/main.py

```diff
@@ -642,6 +646,31 @@ def _start_periodic_project_summaries_calculation():
         )
 
 
+def _start_periodic_partition_management():
+    retention_days = config.crud.alert_activations.retention_days
```

Comment:
Here again, I'd generalize this (the name of this function is already generalized to "partition management" which is great) - iterate over the config values available and call the `_manage_partitions` function for each of them (you'll probably need to pass the pass the table name to the function).
This means that the config keys will have to match the DB table names, but that should be fine.